galaxy/api/internal/views/imports.py

```diff
@@ -97,7 +97,7 @@ def get_repsository_queryset(self):
 
         name = self.request.query_params.get('name')
         if name:
-            qs = qs.filter(repository__name__iexact=name)
+            qs = qs.filter(repository__name__icontains=name)
```

Comment:
This can change the result from a single collection to multiple, which doesn't seem right.

Reply:
The problem is that we use that we need to use that query parameter for the filter which means that `iexact` doesn't return anything when you type a partial match into the filter box. I thought that was a really frustrating user experience so I switched it to `icontains`. If we want to guarantee that we are only getting one repo or collection back, I think we should probably add an ID filter as well.